app/scripts/app.config.js

```diff
@@ -246,15 +246,23 @@ angular.module('confRegistrationWebApp')
               $cookies.put('crsAuthProviderType', '');
               $cookies.put('crsToken', $route.current.params.token);
               $rootScope.crsToken = $cookies.get('crsToken');
-              ProfileCache.getCache().then(function (data) {
-                $cookies.put('crsAuthProviderType', data.authProviderType);
-                if(angular.isDefined($cookies.get('regType'))) {
-                  $location.path($cookies.get('intendedRoute') || '/').search('regType', $cookies.get('regType')).replace();
-                  $cookies.remove('regType');
-                } else {
-                  $location.path($cookies.get('intendedRoute') || '/').replace();
-                }
-              });
+              ProfileCache.getCache()
+                .then(function (data) {
+                  $cookies.put('crsAuthProviderType', data.authProviderType);
+                })
+                .catch(() => {
+                  $cookies.remove('crsToken');
+                  $cookies.remove('crsToken', { domain: 'eventregistrationtool.com'} );
+                  $cookies.remove('crsToken', { domain: 'www.eventregistrationtool.com'} );
```

Comment:
These extra deletes are from https://github.com/CruGlobal/conf-registration-web/pull/575 which created extra cookies. It doesn't seem like browsers load cookies deterministically and we had the wrong cookie loaded.

I've reverted it since then but it left cookies around.